rel/files/ejabberd.cfg

```diff
@@ -665,15 +665,15 @@
 %  {mod_muc_light, [
 %                   {host, "muclight.@HOST@"}
 %                  ]},
-  {mod_muc, [
-             {host, "muc.@HOST@"},
-             {access, muc},
-             {access_create, muc_create}
-            ]},
-  {mod_muc_log, [
-                 {outdir, "/tmp/muclogs"},
-                 {access_log, muc}
-                ]},
+  %{mod_muc, [
+             %{host, "muc.@HOST@"},
+             %{access, muc},
+             %{access_create, muc_create}
+            %]},
+  %{mod_muc_log, [
+                 %{outdir, "/tmp/muclogs"},
+                 %{access_log, muc}
+                %]},
```

Comment:
I would move those comment characters to the beginning of each line but this is minor issue.
For me the PR is ready to merge